rudder-web/src/main/scala/com/normation/rudder/web/rest/node/NodeAPIService4.scala

```diff
@@ -63,13 +64,13 @@ class NodeApiService4 (
 
   def getNodeDetails(nodeId: NodeId, detailLevel: NodeDetailLevel, state: InventoryStatus) = {
     for {
-      node          <- nodeInfoService.getNode(nodeId)
-      nodeInventory <- inventoryRepository.get(nodeId,state)
+      inventory <- inventoryRepository.get(nodeId,state)
+      node      = nodeInfoService.getNode(nodeId).getOrElse(Node(inventory))
```

Comment:
I'm not sure that it is what we want, because node in error also get the defaults information, not only the missing one. 
That's clearly a design problem with getNode that sould return a Box[Option[Node]]. 
In the meantime, could you code it like that:
node <- nodeInfoService.getNode(nodeId) match { case Full(node) => Full(node) ; case eb: EmptyBox => if(inventory /* check status different than accepted */) Full(Node(inventory) else eb }

 ?